cloud/docker_client.go

```diff
@@ -266,6 +268,45 @@ func (c *dockerClientImpl) ListContainers(ctx context.Context, h *host.Host) ([]
 	return containers, nil
 }
 
+// ListImages lists all images on the specified host machine.
+func (c *dockerClientImpl) ListImages(ctx context.Context, h *host.Host) ([]types.ImageSummary, error) {
+	dockerClient, err := c.generateClient(h)
+	if err != nil {
+		return nil, errors.Wrap(err, "Failed to generate docker client")
+	}
+
+	// Get all container images
+	opts := types.ImageListOptions{All: false}
+	images, err := dockerClient.ImageList(ctx, opts)
+	if err != nil {
+		err = errors.Wrap(err, "Docker list API call failed")
+		return nil, err
+	}
+
+	return images, nil
+}
+
+// RemoveImage forcibly removes an image from its host machine
+func (c *dockerClientImpl) RemoveImage(ctx context.Context, h *host.Host, imageID string) error {
+	dockerClient, err := c.generateClient(h)
+	if err != nil {
+		return errors.Wrap(err, "Failed to generate docker client")
+	}
+
+	opts := types.ImageRemoveOptions{Force: true}
```

Comment:
What's the difference if we don't pass the force param?

Reply:
Even if all containers running this image have been stopped, leaving out the force param results in this error: 
`Failed to remove image 'sha256:74f8760a2a8b28abade3fcbcdb6998543f1d9b4a6fb61463c10adc0765c3cb12': Error response from daemon: {"message":"conflict: unable to delete 74f8760a2a8b (must be forced) - image is being used by stopped container ba39421ec78d"}`